core/qthelper.h

```diff
@@ -116,6 +116,21 @@ void moveInVector(Vector &v, int rangeBegin, int rangeEnd, int destination)
 		std::rotate(it + destination, it + rangeBegin, it + rangeEnd);
 }
 
+// Qt overloads some signals(!) which can't therefore be used in connect() calls with
+// pointers-to-member functions where the signatures of signal and slot don't match perfectly.
+// For this case, Qt 5.7 provides the qOverload<> helper.
+// Since we still support Qt 5.5 do our own implementation here. The template arguments
+// are the function parameters of the member function to pick.
+// Usage: "disambiguate<double>::get(&QDoubleSpinBox::valueChanged)".
+template <typename... Args>
+struct disambiguate {
```

Comment:
I would just call it qOverload and wrap it in an #ifdef, so we simply use the Qt version if it is present.

Reply:
Hmm.. I called it differently on purpose to make clear that it is a new implementation, not a copy. Is there any advantage in using Qt's version?

Reply:
It gives us the chance to just drop the new implementation once Qt 5.7 can be a required version. And it makes it clear for every other Qt programmer what this is about. That is is not a copy is IMHO clear for everyone looking at the implementation.

Reply:
I like @DerDakon 's approach here. Call it the same and only use it if we are on a Qt version that's too old (<5.7 I believe). And then, once we decide to drop Ubuntu 16.04 (and other Qt 5.5 based distros) we can simply remove the code again.

Reply:
Fine with me - but obviously I can't test it, because I only have Qt 5.11.1. I'm a bit weary of code that compiles differently for different build environments.

Weirdly, it turns out that it compiles without `#ifdef` even on Qt 5.11. I'm too lazy to find out why. Let's just wait what clang thinks - it seems to be the more picky compiler.

Reply:
Well... Seems to compile in all test environments.

Reply:
> Weirdly, it turns out that it compiles without #ifdef even on Qt 5.11.

I see - because I misspelled it. The template is QOverload, not qOverload. Stupid me.